tech-audit.js

```diff
@@ -268,6 +269,10 @@
     clientSideListAdd("Twitter Widgets");
   }
 
+  if (elementExists('[data-reactroot]') || elementExists('[data-reactid]')) {
+    clientSideListAdd("React");
+  }
```

Comment:
Ooh, this is an approach I hadn’t considered… what versions of React will this work with?